src/cryptoadvance/specter/wallet.py

```diff
@@ -402,6 +402,39 @@ def fetch_transactions(self):
                 self._addresses.add(addresses, check_rpc=False)
                 self._addresses.add(change_addresses, check_rpc=False)
 
+    def import_electrum_label_export(self, electrum_label_export, check_rpc=False):
+        import json
```

Comment:
Unnecessary import, `json` is already imported at the top of the file

Reply:
Thanks. I removed it.